### PR TITLE
When a user will try to save incorrect color he will be notified and the color won't be saved

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/4972

## Description
When a user will try to save incorrect color he will be notified and the color won't be saved

## Screenshots/screencasts
Error-handler (2)

## Backward compatibility

This change is fully backward compatible.

### CODE REVIEW

File: src/components/fields/ColorField.vue [hunk 1]
```diff
@@ -141,14 +146,57 @@ export default {
...
 
-      if (color === this.valueToShow) {
+      if (color === this.valueToShow || !this.isValid) {
         return
       }
 
       sendCssToFrame(color, this.data.fieldConfig)
     },
+    validateColor() {
+      let errorsFields = document.querySelectorAll('input.error')
```
Comment (squallstar) [inline src/components/fields/ColorField.vue]: you don't seem to be reassigning this value so this should be a `const`

File: src/scss/base/validation.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+input {
```
Comment (squallstar) [inline src/scss/base/validation.scss]: I believe we use two spaces for indentation

File: src/scss/base/validation.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    &.error, &.error:hover, &.error:active, &.error:focus {
+        border: 1px solid #eb4034 !important;
```
Comment (squallstar) [inline src/scss/base/validation.scss]: is `!important` required here?

Comment (YaroslavOvdii) [reply src/scss/base/validation.scss]: @squallstar 
Yes, there is a color picker library class which is rewrites this rule. Therefore we need to use `!important`

File: src/components/fields/ColorField.vue (hunk 1 above)
Comment (YaroslavOvdii) [reply src/components/fields/ColorField.vue]: @squallstar fixed

File: src/scss/base/validation.scss (hunk 2 above)
Comment (YaroslavOvdii) [reply src/scss/base/validation.scss]: @squallstar fixed

**Pull request merged**